go.mod

```diff
@@ -4,8 +4,8 @@ go 1.15
 
 require (
 	github.com/francoispqt/gojay v1.2.13
-	github.com/prometheus/client_golang v1.9.0
-	github.com/prometheus/common v0.15.0
+	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/common v0.26.0
 	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.5.1
 	k8s.io/apimachinery v0.19.0
```

Comment:
I'm not sure if you'd like to bump this to v0.22.x as well.

Reply:
@Huang-Wei Please update this one too

Reply:
ok, k8s v1.22.3 should be good, which vendors the following:

```
	github.com/prometheus/client_golang v1.11.0
	github.com/prometheus/client_model v0.2.0
	github.com/prometheus/common v0.26.0
```